### ISSUE TITLE
High CPU usage on custom trained model (car_exists)

### ISSUE THREAD

**Issue by elechocruz** [PaddlePaddle/PaddleClas#2638]
Python: 3.8.10
paddlepaddle: 2.3.2
paddleclas: 2.5.1
Windows 10 Pro
CPU: Intel i7 11th 1165G7 2.8Ghz
Backbone: PPCLNet_x1_0
Command: python -m paddle.distributed.launch  tools/train.py  -c ./ppcls/configs/PULC/car_exists/PPLCNet_x1_0.yaml

Hello, I'm trying to train a model to find out if a car lane is occupied or not, so I took the car_exists model as a base, which works quite well, but it's not 100% adequate for my needs, so I decided to train my own model.

The problem is that once my model was trained, the CPU usage got too high, it went from ~8% with the car_exists model to ~50% with my model. With the example model I also have a fairly high CPU consumption, but if I add the parameter cpu_num_threads=1 it is solved, this does not work if I do it with the model trained by me.

**EDIT: I test my model on a Alma Linux Machine and the CPU usage is low too, like ~7% so, this issue is only presented on Windows**

The model was trained with aproximately 100 images for ocuped and 100 images for empty, and it gives me good predictions and good performance because it only takes ~9ms in make the prediction, meanwhile the car_exists model takes ~40 ms in do the prediction.

The app is simple, just take one frame from a RTSP stream and check if a truck exists or not
This is my CPU when run the app with my model.
![image](https://user-images.githubusercontent.com/54409459/217678510-7fa93d3c-bdd1-4349-95ab-6048e79bfcb1.png)

And this is when use the car_exists model.
![image](https://user-images.githubusercontent.com/54409459/217678721-cf9bda3a-2b09-455d-bf22-06b475899f05.png)

Another thing that I have noticed is that when I load the trained model the following lines appear that do not appear when I load the car_exists model

e[37m---    fused 0 elementwise_add with relu activatione[0m
e[37m---    fused 0 elementwise_add with tanh activatione[0m
e[37m---    fused 0 elementwise_add with leaky_relu activatione[0m
e[37m---    fused 0 elementwise_add with swish activatione[0m
e[37m---    fused 0 elementwise_add with hardswish activatione[0m
e[37m---    fused 0 elementwise_add with sqrt activatione[0m
e[37m---    fused 0 elementwise_add with abs activatione[0m
e[37m---    fused 0 elementwise_add with clip activatione[0m
e[37m---    fused 0 elementwise_add with gelu activatione[0m
e[37m---    fused 0 elementwise_add with relu6 activatione[0m
e[37m---    fused 0 elementwise_add with sigmoid activatione[0m
e[37m---    fused 0 elementwise_sub with relu activatione[0m
e[37m---    fused 0 elementwise_sub with tanh activatione[0m
e[37m---    fused 0 elementwise_sub with leaky_relu activatione[0m
e[37m---    fused 0 elementwise_sub with swish activatione[0m
e[37m---    fused 0 elementwise_sub with hardswish activatione[0m
e[37m---    fused 0 elementwise_sub with sqrt activatione[0m
e[37m---    fused 0 elementwise_sub with abs activatione[0m
e[37m---    fused 0 elementwise_sub with clip activatione[0m
e[37m---    fused 0 elementwise_sub with gelu activatione[0m
e[37m---    fused 0 elementwise_sub with relu6 activatione[0m
e[37m---    fused 0 elementwise_sub with sigmoid activatione[0m
e[37m---    fused 0 elementwise_mul with relu activatione[0m
e[37m---    fused 0 elementwise_mul with tanh activatione[0m
e[37m---    fused 0 elementwise_mul with leaky_relu activatione[0m
e[37m---    fused 0 elementwise_mul with swish activatione[0m
e[37m---    fused 0 elementwise_mul with hardswish activatione[0m
e[37m---    fused 0 elementwise_mul with sqrt activatione[0m
e[37m---    fused 0 elementwise_mul with abs activatione[0m
e[37m---    fused 0 elementwise_mul with clip activatione[0m
e[37m---    fused 0 elementwise_mul with gelu activatione[0m
e[37m---    fused 0 elementwise_mul with relu6 activatione[0m
e[37m---    fused 0 elementwise_mul with sigmoid activatione[0m

There is my yaml file used to training 
# global configs
Global:
  checkpoints: null
  pretrained_model: null
  output_dir: ./output/
  device: cpu
  save_interval: 1
  eval_during_train: True
  eval_interval: 1
  start_eval_epoch: 10
  epochs: 20
  print_batch_step: 10
  use_visualdl: False
  # used for static mode and model export
  image_shape: [3, 224, 224]
  save_inference_dir: ./inference
  # training model under @to_static
  to_static: False
  use_dali: False

# model architecture
Arch:
  name: PPLCNet_x1_0
  class_num: 2
  pretrained: True
  use_ssld: True
  use_sync_bn: True
 
# loss function config for traing/eval process
Loss:
  Train:
    - CELoss:
        weight: 1.0
  Eval:
    - CELoss:
        weight: 1.0

Optimizer:
  name: Momentum
  momentum: 0.9
  lr:
    name: Cosine
    learning_rate: 0.0125
    warmup_epoch: 5
  regularizer:
    name: 'L2'
    coeff: 0.00004

# data loader for train and eval
DataLoader:
  Train:
    dataset:
      name: ImageNetDataset
      image_root: ./dataset/car_exists/
      cls_label_path: ./dataset/car_exists/train_list.txt
      transform_ops:
        - DecodeImage:
            to_rgb: True
            channel_first: False
        - RandCropImage:
            size: 192
        - RandFlipImage:
            flip_code: 1
        - TimmAutoAugment:
            prob: 0.5
            config_str: rand-m9-mstd0.5-inc1
            interpolation: bicubic
            img_size: 192
        - NormalizeImage:
            scale: 1.0/255.0
            mean: [0.485, 0.456, 0.406]
            std: [0.229, 0.224, 0.225]
            order: ''
        - RandomErasing:
            EPSILON: 0.5
            sl: 0.02
            sh: 1.0/3.0
            r1: 0.3
            attempt: 10
            use_log_aspect: True
            mode: pixel
    sampler:
      name: DistributedBatchSampler
      batch_size: 64
      drop_last: False
      shuffle: True
    loader:
      num_workers: 8
      use_shared_memory: True

  Eval:
    dataset: 
      name: ImageNetDataset
      image_root: ./dataset/car_exists
      cls_label_path: ./dataset/car_exists/val_list.txt
      transform_ops:
        - DecodeImage:
            to_rgb: True
            channel_first: False
        - ResizeImage:
            resize_short: 256
        - CropImage:
            size: 224
        - NormalizeImage:
            scale: 1.0/255.0
            mean: [0.485, 0.456, 0.406]
            std: [0.229, 0.224, 0.225]
            order: ''
    sampler:
      name: DistributedBatchSampler
      batch_size: 64
      drop_last: False
      shuffle: False
    loader:
      num_workers: 4
      use_shared_memory: True

Infer:
  infer_imgs: deploy/images/PULC/car_exists/objects365_00001507.jpeg
  batch_size: 10
  transforms:
    - DecodeImage:
        to_rgb: True
        channel_first: False
    - ResizeImage:
        resize_short: 256
    - CropImage:
        size: 224
    - NormalizeImage:
        scale: 1.0/255.0
        mean: [0.485, 0.456, 0.406]
        std: [0.229, 0.224, 0.225]
        order: ''
    - ToCHWImage:
  PostProcess:
    name: ThreshOutput
    threshold: 0.9
    label_0: no_car
    label_1: contains_car

Metric:
  Train:
    - TopkAcc:
        topk: [1, 2]
  Eval:
    - TprAtFpr:
        max_fpr: 0.01
    - TopkAcc:
        topk: [1, 2]

And there is some examples of my dataset

Occuped
![camion(46)](https://user-images.githubusercontent.com/54409459/217678120-009f66f2-6ed2-4299-86e5-c75576e3d79d.jpeg)

Empty
![nada(33)](https://user-images.githubusercontent.com/54409459/217678189-7b7a7add-a347-4835-8cdc-91bf9d65ddc9.jpeg)
